pandas/core/generic.py

```diff
@@ -457,12 +457,39 @@ def axes(self):
 
     @property
     def ndim(self):
-        """Number of axes / array dimensions"""
+        """
+        Return an int representing the number of axes / array dimensions.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+        >>> df.ndim
+        2
+
+        >>> df = pd.DataFrame({'col1': [1, 2, 3], 'col2': [4, 5, 6],
+        ...                    'col3': [7, 8, 9]})
+        >>> df.ndim
+        2
+        """
         return self._data.ndim
 
     @property
     def size(self):
-        """number of elements in the NDFrame"""
+        """
+        Return a numpy.int64 representing the number of elements
+        in this object.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+        >>> df.size
+        4
+
+        >>> df = pd.DataFrame({'col1': [1, 2, 3], 'col2': [4, 5, 6],
+        ...                    'col3': [7, 8, 9]})
+        >>> df.size
+        9
```

Comment:
The same here, can you also add an example for Series? And I think it would be useful to put a small sentence before the example explaining the result (so in case of dataframe number of rows times number of columns)